src/js/features/progress.js

```diff
@@ -102,17 +118,18 @@ Object.assign(MediaElementPlayer.prototype, {
 
 					pos = x - offset.left;
 					percentage = (pos / width);
-					newTime = (percentage <= 0.02) ? 0 : percentage * media.duration;
+					t.newTime = (percentage <= 0.02) ? 0 : percentage * media.duration;
 
-					// seek to where the mouse is
-					if (mouseIsDown && newTime.toFixed(4) !== media.currentTime.toFixed(4)) {
-						media.setCurrentTime(newTime);
-					}
+					// fake seek to where the mouse is 
+					if (mouseIsDown && t.newTime.toFixed(4) !== media.currentTime.toFixed(4)) { 
+						t.setCurrentRailHandle(t.newTime);
```

Comment:
No is should not be added , thats the whole point of doing the changes. 

In average/low memory pc when the player is added along with huge amounts of UI elements there is a (lag/delay OR no response) of `t.current/t.handle` when calling `handleMouseMove` because of the   `media.setCurrentTime(t.newTime);` because mousemove is expensive event. 

If you look at famous players like youtube/vimeo etc they dont setTIme of video on mousemove rather on mouseup. 

One thing that i can say is, by creating a `timeout` function on `mousedown touchstart` to do the  media.setCurrentTime(t.newTime); which clears `clearTimeout` on handleMouseMove will do the job.